modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java

```diff
@@ -44,6 +44,9 @@
 
 /**
  * Defines supported features and check its on other nodes.
+ *
+ * Note: for creating new feature and assigning ID, please update
```

Comment:
More articles to the God of articles. Note: for creating **a** new feature and assigning **an** ID, please update **the** page

Reply:
done.